cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -15,10 +15,20 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 
 == New and changed features
 
-* View high-level information about your virtual machines by
-using the *Virtual Machines* dashboard in the {product-title} web console.
-Access it by navigating to the *Workloads* -> *Virtual Machines* page and
-selecting a virtual machine.
+* Enhancements to workflow for importing a single VMware virtual machine.
+
+* Edit Virtual machine templates. Previously, only virtual machines could be edited.
+
+* Health monitoring of Virtual Machine-backed services, equivalent to that for Pod-based services.
```

Comment:
s/Virtual Machine/virtual machine

I don't think "equivalent to that for" is the best way to phrase this.

Reply:
Rephrased.